Realm/RLMQueryUtil.mm

```diff
@@ -282,6 +346,10 @@ void add_link_constraint_to_query(realm::Query & query,
     }
 }
 
+void add_link_constraint_to_query(realm::Query& query, NSPredicateOperatorType operatorType, RLMObject *obj, NSUInteger column) {
```

Comment:
I had to look at the code a bit to figure out why this is needed, so a brief comment would be useful.